app/javascript/styles/kkt-angely/_mixins.scss

```diff
@@ -0,0 +1,23 @@
+@mixin panel-format($opacity:1) {
+  background: $panel-background;
+  border-radius: 8px;
+  border: solid 2px $panel-border;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.4);
+}
+
+@mixin status-position() {
+  margin-left: 64px;
+  margin-right: 2px;
+  padding-left: 8px;
+  margin-bottom: 8px;
+}
+
+@mixin avatar-radius() {
+  border-radius: 4px;
+  background: transparent no-repeat;
+  background-position: 50%;
+  background-clip: padding-box;
+  border: solid 2px $panel-border;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.4);
+}
+
```

Comment:
looks like this file (and probably some of the others) are the exact same between all the kkt styles -- possible to just use the existing import rather than duplicating it across multiple files?

I imagine the only difference between the styles is the colors (rather than the padding/margins and things like that), so from a maintenance perspective, I think it would be nice to have them all import from some common kkt style and just have overrides for the values that are actually different.